maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -41,6 +41,11 @@ internal class MarkerNode(
     }
 }
 
+@Immutable
+enum class InfoWindowVisibility {
+    VISIBLE, HIDDEN
```

Comment:
```suggestion
    SHOWN, HIDDEN
```
I recommend that the chosen verbs here match that of the original API.